x-pack/plugins/apm/server/lib/services/get_throughput.ts

```diff
@@ -107,7 +101,12 @@ export async function getThroughput({
     response.aggregations?.timeseries.buckets.map((bucket) => {
       return {
         x: bucket.key,
-        y: bucket.throughput.value,
+        y: calculateThroughput({
+          start,
+          end,
+          value: bucket.doc_count,
+          unit: throughputUnit,
+        }),
```

Comment:
You are using the interval (`end - start`) of the of the entire time range. You should use the bucket interval instead